src/chaos-experiments/models/chaosExperimentsManager.js

```diff
@@ -89,9 +93,10 @@ module.exports.insertChaosJobExperiment = async (jobExperimentId, jobId, experim
     await databaseConnector.insertChaosJobExperiment(jobExperimentId, jobId, experimentId, startTime, endTime, contextId);
 };
 
-module.exports.runChaosExperiment = async (kubernetesChaosConfig, jobExperimentId) => {
+module.exports.runChaosExperiment = async (kubernetesChaosConfig, jobId, jobExperimentId) => {
     try {
-        await connector.runChaosExperiment(kubernetesChaosConfig);
+        const mappedResource = buildExperimentResource(kubernetesChaosConfig, jobId);
```

Comment:
Can you rename to mappedKubernetesChaosConfig?

Reply:
done